feat(cli): introduce `planet key derive` (#1268)
planetarium/libplanet: CHANGES.md

    specific peer. [[#1240]]
 -  Added the option `--no-passphrase` to `planet key remove` command to remove
    key without asking passphrase.  [[#1213], [#1265]]

>  -  Added `planet key convert private-key` subcommand to convert private key
>     into address or public key. [[#1268]]

Reviewer: How about using the term “derive”, which is conceptually more accurate, instead of “convert”?

```suggestion
 -  Added `planet key derive PRIVATE-KEY` subcommand to derive the address or
    public key from a private key.  [[#1268]]
```

Author: I applied your suggestion. Could you check it? The commits are a35c800, 05b04c3, 7517a10.

>  -  Added `planet key derive PRIVATE-KEY` subcommand to derive the address or
>     public key from a private.  [[#1268]]

Reviewer: For consistency… (We've never shown metavariables for CLI commands/subcommands in this document.)

```suggestion
 -  Added `planet key derive` subcommand to derive the address or
    public key from a private.  [[#1268]]
```
